### ISSUE TITLE
[Bug]: Issues displaying homepage and settings page

### ISSUE THREAD

**Issue by thombruce** [thombruce/tnt-2024#70]
### What happened?

Homepage shows "Page not found" even though `0.index.md` exists and has content.

Settings page shows nothing - it shows even less if refreshed, stuck in a perpetual loading cycle which never resolves.

### Version

0.10.x (Default)

### Relevant log output

```shell
TypeError: Cannot read properties of undefined (reading 'map')
    at tL47dXQB.js:1:639
    at Si.fn (BcULtgx3.js:10:9144)
    at Si.run (BcULtgx3.js:10:1517)
    at get value (BcULtgx3.js:10:9389)
    at ae (BcULtgx3.js:10:10316)
    at Proxy.<anonymous> (tL47dXQB.js:1:1031)
    at Go (BcULtgx3.js:14:3352)
    at Si.k [as fn] (BcULtgx3.js:14:39420)
    at Si.run (BcULtgx3.js:10:1517)
    at p.update (BcULtgx3.js:14:39710)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by thombruce** [comment]
Settings page fixed: d8c93c0c8c31de688b0f918848b7279dbc3f1bd7

Homepage (on thombruce.com) is gonna be a little different since it didn't feature a form. Still looking into that one.

Problem does also exist on tnt.thombruce.com but is not replicated on dev. This exists purely on the live sites... however I have been suppressing errors to overcome an earlier issue. Perhaps if I attempted a local build, I could find an issue.

**Comment by thombruce** [comment]
Issue resolved as of a08839ee1480d25b659ea7efe52b6ca076716c37 after changing how an improper Lodash import was handled.
